show/reboot_cause.py

```diff
@@ -23,6 +24,100 @@ def read_reboot_cause_file():
 
     return reboot_cause_dict
 
+# Function to fetch reboot cause data from database
+def fetch_reboot_cause_from_db(module_name):
+    table = []
+    r = []
+    wrapper = textwrap.TextWrapper(width=30)
+
+    # Read the previous reboot cause
+    if module_name == "all" or module_name == "SWITCH":
+        reboot_cause_dict = read_reboot_cause_file()
+        reboot_cause = reboot_cause_dict.get("cause", "Unknown")
+        reboot_user = reboot_cause_dict.get("user", "N/A")
+        reboot_time = reboot_cause_dict.get("time", "N/A")
+
+        r.append("SWITCH")
+        r.append(reboot_cause if reboot_cause else "")
```

Comment:
There is no need to have an if statement in all 3 cases because `reboot_cause`, `reboot_user`, and `reboot_time` have a default value.

Reply:
Yes